lambda_service/insert_data.py

```diff
@@ -94,201 +93,34 @@ def main(event, context):
     # The file_extension signifies the filetype, such as 'fits' or 'txt'.
     file_extension = file_key.split('.')[1]
     
-    # The site is derived from the beginning of the base filename (eg. 'WMD')
+    # The site is derived from the beginning of the base filename (eg. 'wmd')
     site = base_filename[0:3] 
     
-    print(f"file_path: {file_path}")
-    print(f"base filename: {base_filename}")
-    print(f"data_type: {data_type}")
-    print(f"file_extension: {file_extension}")
-    print(f"site: {site}")
-
-
-    # Set user_id
-    user_id = 180
-  
-    if file_extension == "txt":
-            
+    logger.info(f"file_path: {file_path}")
```

Comment:
I like to keep my logs single-line just to avoid log spam. If you can condense these 5 distinct bits of information into a single line, it might be better. These also feel kind of like debug logs, but I am not totally sure.